x/launch/types/expected_keepers.go

```diff
@@ -5,6 +5,7 @@ import (
 )
 
 type ProfileKeeper interface {
+	HasCoordinator(ctx sdk.Context, id uint64) bool
```

Comment:
I would instead returns `(address string, found bool)` in `GetCoordinatorAddressFromID` to make both methods consistent

And btw I think we can rename `GetCoordinatorAddressFromID` into `CoordinatorAddressFromID` for consistency as well